hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/io/storage/row/parquet/ParquetRowDataWriter.java

```diff
@@ -31,6 +31,7 @@
 import org.apache.flink.table.types.logical.RowType;
 import org.apache.flink.table.types.logical.TimestampType;
 import org.apache.flink.util.Preconditions;
+import org.apache.hudi.common.util.ValidationUtils;
 import org.apache.parquet.io.api.Binary;
```

Comment:
You might need to fix your import sequence of the IDEA.

Reply:
Hmmm, I don't think there's anything wrong with this, `h` is after `f`, so, `org.apache.hudi` should be behind `org.apache.flink` right?

Reply:
org.apache.hudi package should be alway in the highest import sequence, you many need to import the checkstyle from the file: https://github.com/apache/hudi/blob/d9b29e540314f842d8d3aa86f8429cca8a8cf786/style/checkstyle.xml#L291

Reply:
Take other Java files for reference of import sequence.

Reply:
Got it, will fix that.

Reply:
Man, after adding the hudi checkstyles, it's adding whitelines instead of ordering the imports...